README.md

```
@@ -81,6 +81,122 @@ If you are testing codegen it's important to have SPARK_TESTING set to yes, as w
+libraryDependencies ++= Seq(
+  "org.apache.spark" %% "spark-connect"            % sparkVersion % Test,
+  "org.apache.spark" %% "spark-connect-client-jvm" % sparkVersion % Test
+)
+```
+
+### Pointing at an already-running Connect server
+
+Set either of these and the suite connects to that server instead of starting
+one of its own:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)